test/cat.jl

```diff
@@ -41,8 +40,8 @@ module TestCat
         dt = DataTable()
         DataTables.hcat!(dt, NullableCategoricalVector(1:10))
         @test isequal(dt[1], NullableCategoricalVector(1:10))
-        DataTables.hcat!(dt, NullableArray(1:10))
-        @test isequal(dt[2], NullableArray(1:10))
+        DataTables.hcat!(dt, Vector(1:10))
+        @test isequal(dt[2], Vector(1:10))
```

Comment:
BTW, it's not a requirement, but everytime `NullableVector` is removed, `isequal` can be replaced with `==`, which is more standard and easier to read.